package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wp-dm-custom-embed",
-  "version": "1.2.1",
+  "version": "1.5.0-1",
```

Comment:
so, this is autoversioning?

Reply:
For that not yet. I still manually set it up. It needs a base version before actually using it.

Reply:
I think there's a command such as `npm version patch` I forget.